### PR TITLE
[RA1 Ch02] APIs: requirement added per service

### LINKED ISSUE #1621: [RA1- Ch02] APIs: requirements for all OpenStack Mandatory Services APIs
In section 2.3.4 "Interfaces & APIs requirements", we should add a dedicated requirement for each OpenStack mandatory service. Currently, only "Horizon" and "Placement" have a dedicated requirement and req.int.api.01 is too generic to cover all the other services

### PR DESCRIPTION
**Pull request by karinesevilla**
fixes #1621 
Section 2.3.4 APIs
Requirements added for each OpenStack Mandatory Service

### CODE REVIEW

**Comment by pgoyal01** [comment]
@karinesevilla what is the process for deciding the mandatory capabilities in RA-1 Ch05? Maybe we need a review with a wider set of reviewers on these capabilities in RA-1 Ch05.

**Comment by collivier** [comment]
@pgoyal01 Karine clarified the existing work.
Chap5, RC and Functest are synced. Everyone is free to modify the tabs in one way (optional -> mandatory) or the other (mandatory -> optional). In that case, RC and Functest must be aware of any change to avoid mismatches (Ideally all should be changed in one PR) to keep the traceability in place.

**Comment by collivier** [comment]
req.inf.stg.06 (glance), req.inf.stg.11 (cinder) and  req.inf.ntw.18 (neutron) are less precised than this one and must be removed or fixed (false "should" requirements)

**Comment by karinesevilla** [comment]
@pgoyal01 I agree with you that the chapter 5 must be reviewed by more reviewers, as the capabilties detailed in chapter 5 will by part of conformance process.

**Comment by karinesevilla** [comment]
@iangardner22 To remove CNTT, we can replace "CNTT conformance" by "Infrastructure conformance"?

**Comment by rabi-abdel** [comment]
@pgoyal01 please review

**Comment by pgoyal01** [comment]
@karinesevilla This is making the "mandatory features selected for the CNTT conformance" as the APIs. Instead of conformance driven by RA requirements and specifications driving the conformance, we are having conformance drive the requirements. I think these requirement changes are not aligned with the CNTT principles and policies.

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -115,11 +115,16 @@ Traceability to Reference Model.
...
-| `req.int.api.02` | API | The Architecture **must** provide GUI access to tenant facing cloud platform core services. | [RA-1 4.3.1.9 "Horizon"](./chapter04.md#4319-horizon) |
-| `req.int.api.03` | API | The Architecture **must** provide APIs needed to discover and manage Cloud Infrastructure resources. | [RA-1 5.2.7. "Placement"](./chapter05.md#527-placement) |
-| `req.int.api.04` | API | The Architecture must expose the latest version and microversion of the APIs for the given CNTT OpenStack release for each of the OpenStack core services | [RA-1 5.2 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#52-core-openstack-services-apis) |
-| `req.int.acc.01` | Acceleration | The Architecture **should** provide an open and standard acceleration interface to VNFs. |  |
+| `req.int.api.01` | API | The Architecture **must** provide APIs to access to the authentication service and all associated mandatory features selected for the CNTT conformance of the given OpenStack release and detailed in chapter 5 | [RA-1 5.2.1 "Keystone"](./chapter05.md#521-keystone) |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:118]:
```suggestion
| `req.int.api.01` | API | The Architecture **must** provide APIs to access to the authentication service and all associated mandatory features detailed in chapter 5 | [RA-1 5.2.1 "Keystone"](./chapter05.md#521-keystone) |
```

**Comment by pgoyal01** [comment]
@karinesevilla I suggested change of one of the requirements to the following:

| Ref # | sub-category | Description |  Traceability |
|----|----------|--------------------|------------|
| `req.int.api.01` | API | The Architecture **must** provide APIs to access to the authentication service and the associated mandatory features detailed in chapter 5 | [RA-1 5.2.1 "Keystone"](./chapter05.md#521-keystone) |

If you agree, then all others can similarly be edited. Thanks

File: doc/ref_arch/openstack/chapters/chapter02.md (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.md:118]: @pgoyal01 I agree to remove "CNTT conformance" per your comment. I'll modify consequently the other requirements

**Comment by karinesevilla** [comment]
@iangardner22 and @collivier Modifications made, can you review again?

File: doc/ref_arch/openstack/chapters/chapter02.md (hunk 1 above)
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter02.md]: Very minor but "provide APIs to access to the" - suggest rewording as "provide APIs to access the" i.e. delete the second "to".  This is repeated in other lines in this section.

Comment (iangardner22) [reply doc/ref_arch/openstack/chapters/chapter02.md]: Also CNTT - need to find a way not to use this term as agreed in TSC with @rabi-abdel
